htdocs/install/mysql/tables/llx_webhook_target.sql

```diff
@@ -0,0 +1,36 @@
+-- Copyright (C) ---Put here your own copyright and developer email---
```

Comment:
Rename the file
llx_webhook_target.sql into llx_webhook_target-webhook.sql
So file will be executed only when module is loaded.